cypress/integration/Images/LeadImageTest/index.js

```diff
@@ -0,0 +1,22 @@
+// <reference types="Cypress" />
+import { Given, And, Then } from 'cypress-cucumber-preprocessor/steps';
+
+Then('lead image is visible', () => {
+    cy.get('.centered-set.image-medium').should('be.visible');
+});
+
+And('{string} caption is correct', (caption) => {
+    cy.get('.caption-container').should('contain.text', caption);
+});
+
+And('{string} credit is correct', (credit) => {
+    cy.get('.image-photo-credit').should('contain.text', credit);
+});
```

Comment:
same here - should be tied to the lead image to ensure we are checking the right credit

Reply:
fixed this